### PR TITLE
use proteinChange property of Mutation instead of aminoAcidChange

### PR DESCRIPTION
**Pull request by alisman**
The new api doesn't return aminoAcidChange property on Mutations.   We need to use the property proteinChange instead.

### CODE REVIEW

File: src/shared/lib/oql/accessors.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ export default class accessors {
         if (this.alterationType(d.molecularProfileId) === 'MUTATION_EXTENDED') {
             if (d.mutationType === "fusion") {
                 return null;
-            } else if (d.aminoAcidChange && d.aminoAcidChange.toLowerCase() === "promoter") {
+            } else if (d.proteinChange && d.proteinChange.toLowerCase() === "promoter") {
```
Comment (alisman) [inline src/shared/lib/oql/accessors.ts]: @jj @onursumer this is not related to the bug but i decided to change this reference to aminoAcidChange as well.  Let me know if that's right or not.

File: src/shared/lib/oql/accessors.ts [hunk 2]
```diff
@@ -125,7 +125,7 @@ export default class accessors {
 
     public mut_amino_acid_change(d: Mutation) {
         if (this.alterationType(d.molecularProfileId) === 'MUTATION_EXTENDED') {
```
Comment (alisman) [inline src/shared/lib/oql/accessors.ts]: we need to refactor oqlFilter.js to typescript and deal with the null case in some fashion.

File: src/shared/lib/oql/accessors.ts (hunk 1 above)
Comment (onursumer) [reply src/shared/lib/oql/accessors.ts]: @alisman looks right to me

**Comment by jjgao** [comment]
Not sure why travis fails. Please feel free to merge the release if code works. @inodb

**Pull request merged**